applications/services/power/power_service/power.c

```diff
@@ -81,6 +160,10 @@ Power* power_alloc() {
     power->battery_view_port = power_battery_view_port_alloc(power);
     power->show_low_bat_level_message = true;
 
+    //Auto shutdown timer
+    power->auto_shutdown_timer =
```

Comment:
Crashing on `./fbt flash_usb_full`. Must be allocated before callback subscription.